src/handlebars/getinvolved.handlebars

```diff
@@ -71,21 +75,15 @@
         <li><a href="https://github.com/AdoptOpenJDK/openjdk-api" target="_blank">openjdk-api</a> -
             <a href="https://github.com/AdoptOpenJDK/openjdk-api/wiki" target="_blank">(wiki)</a>
             - The API for users to get releases and nightly builds</li>
-        <li><a href="https://github.com/AdoptOpenJDK/openjdk-website-backend" target="_blank">openjdk-website-backend</a> -
-            <a href="https://github.com/AdoptOpenJDK/openjdk-website-backend/wiki" target="_blank">(wiki)</a>
-            - Scripts to integrate GitHub with this website</li>
-        <li><a href="https://github.com/AdoptOpenJDK/openjdk-releases" target="_blank">openjdk-releases</a> -
-            <a href="https://github.com/AdoptOpenJDK/openjdk-releases/wiki" target="_blank">(wiki)</a>
-            - OpenJDK tagged releases</li>
-        <li><a href="https://github.com/AdoptOpenJDK/openjdk-nightly" target="_blank">openjdk-nightly</a> -
-            <a href="https://github.com/AdoptOpenJDK/openjdk-nightly/wiki" target="_blank">(wiki)</a>
-            - Nightly releases</li>
+        <li><a href="https://github.com/AdoptOpenJDK/openjdk-binaries" target="_blank">openjdk-binaries</a> -
+            <a href="https://github.com/AdoptOpenJDK/openjdk-binaries/wiki" target="_blank">(wiki)</a>
+            - AdoptOpenJDK tagged nightlies and releases</li>
         <li><a href="https://github.com/AdoptOpenJDK/openjdk-jdk8u" target="_blank">openjdk-jdk8u</a> -
             <a href="https://github.com/AdoptOpenJDK/openjdk-jdk8u/wiki" target="_blank">(wiki)</a>
             - A clone of the OpenJDK jdk8u forest</li>
-        <li><a href="https://github.com/AdoptOpenJDK/openjdk-jdk9" target="_blank">openjdk-jdk9</a> -
-            <a href="https://github.com/AdoptOpenJDK/openjdk-jdk9/wiki" target="_blank">(wiki)</a>
-            - A clone of the OpenJDK jdk9 forest</li>
+        <li><a href="https://github.com/AdoptOpenJDK/openjdk-jdk11u" target="_blank">openjdk-jdk9</a> -
```

Comment:
Link text should be `openjdk-jdk11u` not `openjdk-jdk9`.

Reply:
Thanks fixed!